Sources/Noora/Components/YesOrNoChoicePrompt.swift

```diff
@@ -0,0 +1,125 @@
+import Foundation
+import Rainbow
+
+class YesOrNoChoicePrompt {
```

Comment:
Same as for multiple choices – does this need to be a class?

Reply:
Nop. I changed it to a `struct`.